app/services/gtfs_graph.rb

```diff
@@ -209,6 +210,7 @@ def load_tl_operators
       stops = routes
         .map { |route| route.serves }
         .reduce(Set.new, :+)
+        .map { |i| find_by_onestop_id(i) }
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.